figure.html

```diff
@@ -398,6 +398,131 @@
+
+<img alt="../images/figureROI.jpg" class="align-center" src="images/figureROI.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<li>
+
+<p class="first">ROI shapes can be edited or moved after adding, and copied and pasted.</p>
+
+<p>Click <span class="bold">OK</span> to save the shapes.</p>
+
+<p>Shapes can be copied in the Draw ROIs window and pasted directly onto panels in the main Figure drawing.</p>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Pasted shapes are not scaled, so if pasted onto panels with a different size or zoom level, they may not fit on the panel.</p>
+</div><!-- End note -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figureROICopy.jpg" class="align-center" src="images/figureROICopy.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p class="first">Rectangular ROIs on one panel can be used to crop other panels to the same area.</p>
+
+<p>Select the panel with the ROI on it.</p>
+
+<p>Click <span class="bold">Copy</span> in the ROIs toolbar in the <span class="bold">Labels</span> tab.</p>
+
+<p>Select the panel to be cropped.</p>
+
+<p>In the <span class="bold">Preview</span> tab, click <span class="bold">Paste</span>.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/figurePasteCrop.jpg" class="align-center" src="images/figurePasteCrop.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li>
+<p class="first">Non-microscopy images such as scans of Western blots or exported charts can be imported to the OMERO server in TIFF, PNG or other format and used in Figure.</p>
```

Comment:
``exported charts`` - not sure if this is the best -> maybe ``charts created in other software, like Excel, exported or saved in an image format such as TIFF or PNG ``